@stephengillie Is the validation error intermittent? I got the same error with a previous PR and it went away when I tried again. If it's not intermittent I'm happy to submit a fix but I can't tell what the error is from the logs:
```
============================================================================== 
Task         : Installation Validation
Description  : Invoke an Azure Function
Version      : 1.220.0
Author       : Microsoft Corporation
Help URL     : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/azure-function
============================================================================== 
POST ***?code=***&
Request body: {
  "operationId": "WinGetSvc-Validation-282188-1",
  "buildId": "159831",
  "projectId": "8b78618a-7973-49d8-9174-4360829d979b",
  "planId": "d0baf112-bbc6-46ae-aab4-a787f77c47b6",
  "jobId": "68997d86-8ece-5ae8-3ac5-ccdbfa458731",
  "timelineId": "d0baf112-bbc6-46ae-aab4-a787f77c47b6",
  "taskInstanceId": "b79910ce-dca5-5e27-ad7f-bd285eecb5a2",
  "repository": "microsoft/winget-pkgs",
  "pipelineType": "ValidationPipeline",
  "pullRequestNumber": "282188",
  "buildSourceBranch": "refs/heads/master"
}
				Response Code: OK
				Response: 

8/8/2025 3:31:55 PM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: Started

8/8/2025 3:32:33 PM
Status: InProgress

8/8/2025 3:33:05 PM
Status: InProgress
Installation Verification
****************************************************************************************
Status: Completed
##[error] Failed
```

--- a/manifests/k/Kubetail/Kubetail/0.7.4/Kubetail.Kubetail.installer.yaml
+++ b/manifests/k/Kubetail/Kubetail/0.7.4/Kubetail.Kubetail.installer.yaml
@@ -11,9 +11,9 @@ NestedInstallerFiles:
 Installers:
   - Architecture: x64
     InstallerUrl: https://github.com/kubetail-org/kubetail/releases/download/cli%2Fv0.7.4/kubetail-windows-amd64.zip
-    InstallerSha256: a9c3c479a9a58d7246d1bab99094d02988accf2363b7c585f461a5bc82e9b80f
+    InstallerSha256: 686033af128e9adaf8e0e12851015476ca0caf18a14100d664237bcd8b3fdc18
   - Architecture: arm64
     InstallerUrl: https://github.com/kubetail-org/kubetail/releases/download/cli%2Fv0.7.4/kubetail-windows-arm64.zip
-    InstallerSha256: 0b14b1822c2513e438a49892ab4bbde5b37eb8147ef8993300f449e8454961dc
+    InstallerSha256: ab5cec3e1b40873f26a6476cebf4dfd3f488b91fcf10f75bc431e4435fae6e03
 ManifestType: installer
 ManifestVersion: 1.10.0

